src/convertToObject.js

```diff
@@ -5,8 +5,24 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
```

Comment:
The variable name `cssObject` is clear, but consider a more descriptive name such as `styleObject` or `styleRulesObject` to convey its purpose more precisely.